library/nats

```diff
@@ -1,5 +1,6 @@
 # maintainer: Derek Collison <derek@apcera.com> (@derekcollison)
 # maintainer: Waldemar Salinas <wally@apcera.com> (@wallyqs)
 
+0.6.6: git://github.com/nats-io/nats-docker@9909ce47584df326aa0089e4a1f71af89e5baea6
 0.6.4: git://github.com/nats-io/nats-docker@59e0d88a686f727b15dda47db5323c3722e6d85a
```

Comment:
Is there a reason to keep this `0.6.4` reference around?  We won't delete the already-built tag from the Hub, but removing it here will remove it from the `Supported` list at the top of the image description.